apple-codesign/src/stapling.rs

```diff
@@ -38,12 +38,6 @@ use {
 /// The record name is derived from the digest of the code directory of the
 /// main binary within the bundle.
 pub fn record_name_from_app_bundle(bundle: &DirectoryBundle) -> Result<String, AppleCodesignError> {
```

Comment:
The change looks good to me, but we may want to rename the function to `record_name_from_executable_bundle` or something, since it now works on more than just app bundles.

Reply:
sorted!